From 8a989967597e8fd0b79e14aef006c868c194a958 Mon Sep 17 00:00:00 2001
From: Vincent Wilms <vwilms@outlook.com>
Date: Sat, 11 May 2019 09:53:55 +0200
Subject: [PATCH] Tune-SOEM

---
 CMakeLists.txt      | 2 +-
 soem/ethercattype.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d74604b..7474b5d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -9,7 +9,7 @@ endif()
 
 set(SOEM_INCLUDE_INSTALL_DIR include/soem)
 set(SOEM_LIB_INSTALL_DIR lib)
-set(BUILD_TESTS TRUE)
+set(BUILD_TESTS FALSE)
 
 if(WIN32)
   set(OS "win32")
diff --git a/soem/ethercattype.h b/soem/ethercattype.h
index 86df704..23535a9 100644
--- a/soem/ethercattype.h
+++ b/soem/ethercattype.h
@@ -25,7 +25,7 @@ extern "C"
 
 /** define EC_VER1 if version 1 default context and functions are needed
  * comment if application uses only ecx_ functions and own context */
-#define EC_VER1
+//#define EC_VER1
 
 #include "osal.h"
 
-- 
2.21.0.windows.1

